airflow/www/static/js/grid/index.jsx

```diff
@@ -42,14 +42,19 @@ const myCache = createCache({
 const mainElement = document.getElementById('react-container');
 shadowRoot.appendChild(mainElement);
 
+const now = new Date();
+const minAgo = now.setMinutes(now.getMinutes() - 6);
+
 const queryClient = new QueryClient({
   defaultOptions: {
     queries: {
+      notifyOnChangeProps: 'tracked',
       refetchOnWindowFocus: false,
       retry: 1,
       retryDelay: 500,
-      staleTime: 5 * 60 * 1000, // 5 minutes
       refetchOnMount: true, // Refetches stale queries, not "always"
+      staleTime: 5 * 60 * 1000, // 5 minutes
+      initialDataUpdatedAt: minAgo, // make sure initial data is already expired
```

Comment:
I think this also works, and makes it a little simpler:
```suggestion
      initialDataUpdatedAt: new Date().setMinutes(-6), // make sure initial data is already expired
```